specification/dimensions/commitmentutilization.md

```diff
@@ -0,0 +1,37 @@
+# Commitment Utilization
+
+Commitment Utilization indicates whether a charge utilized a commitment during the charge period. Commitment Utilization applies to all commitments and commitment types applied to the current charge. This includes but is not limited to capacity commitments and commitment-based discounts represented by the Commitment Discount ID. Commitment Utilization is commonly used to identify wastage when a commitment has not been used and to calculate commitment utilization.
+
+The CommitmentUtilization column MUST be present and MUST NOT be null when a commitment is applied or CommitmentDiscountId is set. This column is of type String and MUST be one of the allowed values.
+
+## Column ID
+
+CommitmentUtilization
+
+## Display Name
+
+Commitment Utilization
+
+## Description
+
+Indicates whether a charge utilized a commitment during the charge period.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | True           |
+| Value format    | list-of-values |
+
+Allowed values:
```

Comment:
What is the real value of this column for practitioners especially when it's partially used? Wouldn't be more helpful to display in this column the unused portion of commitment so practitioners could easily calculate the 'waste' across all the commitments. And to display 0 if it's fully used/utilized.
What is suggested value in case of commitment is partially used?

Reply:
The intent isn't to put a number here. It's to identify whether this row was for `Used` commitment or `Not Used` commitment. So if you want to calculate utilization, you can group by this column to get what was used vs. not used. You can also quantify waste by filtering on `Not Used`.

If that's not clear, we should tweak the text to make it clearer. We'll also cover it in the Discount Handling spec, but let's make sure it's clear here first. Any thoughts on how we could make it more obvious?

Reply:
After getting into more details it's clear. Probably the name is a bit misleading which I believe is a subject for discussion anyway.